trin-bridge/src/bridge.rs

```diff
@@ -40,19 +37,42 @@ use trin_validation::accumulator::MasterAccumulator;
 use trin_validation::constants::{EPOCH_SIZE as EPOCH_SIZE_USIZE, MERGE_BLOCK_NUMBER};
 use trin_validation::oracle::HeaderOracle;
 
+// todo: calculate / test optimal saturation delay
+// todo: single block option
+const HEADER_SATURATION_DELAY: u64 = 10; // seconds
+const LATEST_BLOCK_POLL_RATE: u64 = 5; // seconds
+const EPOCH_SIZE: u64 = EPOCH_SIZE_USIZE as u64;
+const FUTURES_BUFFER_SIZE: usize = 32;
+
 pub struct Bridge {
     pub mode: BridgeMode,
     pub portal_clients: Vec<HttpClient>,
+    pub pandaops: PandaOpsMiddleware,
     pub header_oracle: HeaderOracle,
     pub epoch_acc_path: PathBuf,
 }
 
-// todo: calculate / test optimal saturation delay
-// todo: single block option
-const HEADER_SATURATION_DELAY: u64 = 10; // seconds
-const LATEST_BLOCK_POLL_RATE: u64 = 5; // seconds
-const EPOCH_SIZE: u64 = EPOCH_SIZE_USIZE as u64;
-const FUTURES_BUFFER_SIZE: usize = 32;
+impl Bridge {
+    pub fn new(
+        mode: BridgeMode,
+        portal_clients: Vec<HttpClient>,
+        header_oracle: HeaderOracle,
+        epoch_acc_path: PathBuf,
+    ) -> Self {
+        Self {
+            mode,
+            portal_clients,
+            pandaops: PandaOpsMiddleware::new(
```

Comment:
Could we `impl Default` for `PandaOpsMiddleware` instead of using `new` here?

Reply:
Default here would require Default implementations for `BridgeMode`, `HeaderOracle`, `epoch_acc_path`, and `HttpClient`. IMO, this will add more overheating to this PR...

Reply:
I'm talking about `PandaOpsMiddleware::default()` not `Bridge::default()`. `PandaOpsMiddleware::default()` should be fairly trivial to implement